rust-src/concordium_base/src/web3id/mod.rs

```diff
@@ -156,6 +177,9 @@ impl<C: Curve, AttributeType: Attribute<C::Scalar> + serde::Serialize> serde::Se
                 });
                 json.serialize(serializer)
             }
+            CredentialStatement::IdentityCredentials { .. } => {
+                todo!()
```

Comment:
I've used the `did:ccd:{network}:id:{idp_index}:{id_index}` format in the JS SDK.